solutions/3074/01.py

```
@@ -0,0 +1,19 @@
+class Solution:
+    def minimumBoxes(self, apple: List[int], capacity: List[int]) -> int:
```

Ruff (0.14.10)

Undefined name `List`

(F821)